eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/GovSpec.hs

```diff
@@ -81,6 +83,123 @@ spec =
             voter
             dummyGaid
             [inject $ GovActionsDoNotExist @era $ Set.singleton dummyGaid]
+      it "Motion of no confidence can be passed" $ do
+        modifyPParams $ \pp ->
+          pp
+            & ppDRepVotingThresholdsL . dvtCommitteeNoConfidenceL .~ 1 %! 2
+            & ppPoolVotingThresholdsL . pvtCommitteeNoConfidenceL .~ 1 %! 2
+            & ppCommitteeMaxTermLengthL .~ EpochInterval 200
+        let
+          getCommittee =
+            getsNES $
+              nesEsL . esLStateL . lsUTxOStateL . utxosGovStateL . cgEnactStateL . ensCommitteeL
+          assertNoCommittee =
```

Comment:
Without explicit `HasCallStack constraint, you won't know which occurrence triggered the failure
```suggestion
          assertNoCommittee :: HasCallStack => ImpTestM era ()
          assertNoCommittee =
```